desktop/core/src/desktop/auth/backend.py

```diff
@@ -273,11 +274,18 @@ class AllowFirstUserDjangoBackend(django.contrib.auth.backends.ModelBackend):
   Allows the first user in, but otherwise delegates to Django's
   ModelBackend.
   """
-  def authenticate(self, username=None, email=None, password=None):
-    if email is not None:
-      username = email
+  def authenticate(self, *args, **kwargs):
+    if sys.version_info[0] > 2:
+      request = args[0]
+    else:
+      request = None
+
+    username = kwargs['username']
+    password = kwargs['password']
+
+    if 'email' in kwargs:
```

Comment:
some tests were failing so now I have to change like this. the error was no keyword 'email' in kwargs @romainr

Reply:
:+1: This is one good day to port the previous `email is not None`